### PR TITLE
Delete Site: vector figures

### PR DESCRIPTION
**Pull request by adambbecker**
Replaces the export site and start over png figures with svg versions.

##### Delete site export

screen shot 2015-12-08 at 1 07 03 pm

`settings/delete-site/*site-url*`

##### Start over

screen shot 2015-12-08 at 1 07 20 pm

`settings/start-over/*site-url*`

---

/cc @rickybanister

### CODE REVIEW

**Comment by rickybanister** [comment]
Thanks for updating those, looks much better. I'm not really familiar how to do it, but I believe there is some process or tool to remove all the metadata from the svgs that Sketch adds.

cc @alternatekev who might know how?

**Comment by adambbecker** [comment]
Cool, thanks @alternatekev I just ran them through https://github.com/svg/svgo which cleaned those svg files up pretty well.

**Comment by kellychoffman** [comment]
Nice. This might be a good opportunity to add some color to the gray and white landscape.

**Comment by adambbecker** [comment]
@kellychoffman: Yeah that could be interesting to try out. I'll keep this PR pretty focused on just vectorizing the current images, but I'll definitely circle back on the color idea in the future.

File: public/images/delete-site/export-content.svg [hunk 1]
```diff
@@ -0,0 +1 @@
...-4.23-1.54-7.175-2.864-9.48-1.71-2.82-3.378-5.21-3.378-8.072 0-3.16 2.394-6.108 5.772-6.108l.47.042c-6.114-5.594-14.278-9.052-23.256-9.052-12.012 0-22.614 6.19-28.77 15.502l2.223.042c3.59 0 9.148-.427 9.148-.427 1.88-.086 2.094 2.605.256 2.86 0 0-1.88.214-3.976.3l12.568 37.284 7.525-22.593-5.344-14.692c-1.88-.085-3.633-.298-3.633-.298-1.84-.128-1.625-2.947.214-2.86 0 0 5.685.425 9.063.425 3.633 0 9.19-.427 9.19-.427 1.838-.086 2.094 2.605.215 2.86 0 0-1.84.214-3.934.3l12.44 36.985 3.422-11.446zM92.01 76.304c-.085-.172-.17-.3-.213-.47l-10.603-29L70.85 76.86c3.12.895 6.37 1.365 9.746 1.365 4.02 0 7.865-.683 11.414-1.92zm19.024-45.44c0 3.5-.643 7.43-2.608 12.342L97.91 73.57c10.216-5.936 17.098-17.04 17.098-29.724 0-5.98-1.495-11.616-4.188-16.528a31.01 31.01 0 0 1 .214 3.546z" fill="#A8BECE"/><path d="M133.263 53.938v37.715L79 113.728V76.014l54.263-22.076zM24.737 53.938v37.715L79 113.728V76.014L24.737 53.938z" fill="#D5E5EB"/><path d="M155.74 70.355L102.527 92.48 78.125 75.12l55.24-21.407 22.374 16.642z" stroke="#FFF" stroke-width="2.4" fill="#D5E5EB"/><path d="M2.08 70.355L55.294 92.48l24.402-17.36-54.582-21.297L2.08 70.355z" stroke="#FFF" stroke-width="2.4" fill="#D5E5EB"/></g></svg>
```
Comment (mtias) [inline public/images/delete-site/export-content.svg]: may be better to inline these in the component?

Comment (adambbecker) [inline public/images/delete-site/export-content.svg]: @mtias: I suppose we could, although I'm not sure necessarily any big benefit of doing so. Having that image tag makes the component a bit more readable to me, but we still could definitely put it inline.

Comment (mtias) [inline public/images/delete-site/export-content.svg]: We'd avoid a request, and I think we may need whitelisting for non image files at the server level.

Comment (adambbecker) [inline public/images/delete-site/export-content.svg]: Ah, cool, that's a good point. I'll switch them out.

Comment (mtias) [inline public/images/delete-site/export-content.svg]: If we think we may be reusing these we may also look into making it simple components like https://github.com/Automattic/wp-calypso/blob/master/client/components/wordpress-logo/index.jsx

And with React 0.14 just pure functional components :)

Comment (adambbecker) [inline public/images/delete-site/export-content.svg]: Yeah I see what you mean and we could definitely do that in the future. I'd rather do it however, at the point we actually find a concrete need for it. Trying to keep this PR as focused as possible.

Comment (mtias) [inline public/images/delete-site/export-content.svg]: Absolutely :)

**Comment by adambbecker** [comment]
@mtias: Cool, inline'd those svgs, removed the outdated images from `public`, rebased, and squashed.

Although after the rebase, CircleCi is now saying:

```
Happiness engineers actions
    1) "before each" hook for "Actions should be an object"

  Happiness engineers Store
    ✓ Store should be an object
    ✓ Store should have method getHappinessEngineers
    Get Happiness Engineers
      ✓ Should return empty array when there are no happiness engineers
      ✓ Should return an array of happiness engineers when there are happiness engineers


  4 passing (2s)
  1 failing

  1) Happiness engineers actions "before each" hook for "Actions should be an object":
     Error: timeout of 2000ms exceeded. Ensure the done() callback is being called in this test.

...

These tests have failed:
/home/ubuntu/wp-calypso/client/lib/happiness-engineers
```

Time to go track whatever that stuff is down.

**Comment by adambbecker** [comment]
Alright, rebased again and that seemed to get pass the test.

**Comment by rralian** [comment]
Looks good! :shipit:

**Comment by mtias** [comment]
:+1:

**Pull request merged**